src/components/ProductsDashboard/AddProduct/ProductForm/ProductForm.js

```diff
@@ -27,24 +33,27 @@ const renderField = ({
 }) => (
   <React.Fragment>
     <input {...input} type={type} />
-    <p>
-      {touched
-        && (error && <span>{error}</span>)}
-    </p>
+    <div>
+      {
+        touched
+        && error
+        && <span>{error}</span>
```

Comment:
minor

I meant, that in such simple cases we can inline it
`{touched && error && <span>{error}</span>}`